src/MyEventEmitter.js

```diff
@@ -1,14 +1,71 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  listeners = {};
+
+  on(event, callback) {
+    if (this.listeners[event]) {
+      this.listeners[event].push(callback);
+    } else {
+      this.listeners[event] = [callback];
+    }
+  }
+
+  once(event, callback) {
+    callback.once = true;
+
+    if (this.listeners[event]) {
+      this.listeners[event].push(callback);
+    } else {
+      this.listeners[event] = [callback];
+    }
+  }
+
+  off(event, callback) {
+    this.listeners[event] = this.listeners[event].filter(
+      (evn) => evn !== callback,
+    );
+  }
+
+  emit(event, ...args) {
+    for (const callback of this.listeners[event]) {
+      callback(...args);
+    }
```

Comment:
The `emit` method assumes that `this.listeners[event]` is always defined. If there are no listeners for the event, this will throw an error. Consider adding a check to ensure `this.listeners[event]` is defined before iterating over it.